assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -103,16 +101,6 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
           maxLength={MIN_MAX.TITLE.MAX}
         />
       </Row>
-      <Row>
-        <ExpandedTextInput
-          placeholder={__('Assistive text', 'web-stories')}
-          value={alt || ''}
-          onChange={(value) => pushUpdate({ alt: value || null })}
-          clear
-          aria-label={__('Edit: Assistive text', 'web-stories')}
-          maxLength={MIN_MAX.ALT_TEXT.MAX}
-        />
-      </Row>
       <Row>
         <Note>{__('Text for visually impaired users.', 'web-stories')}</Note>
```

Comment:
Let's change this to "Description for indexability and accessibility."

See discussion here: https://docs.google.com/spreadsheets/d/1PjaRp4bueQ3Cv14A7I696m0vjGCIX_kN9tIGh-V-Bkc/edit?disco=AAAAHST_gmM

Reply:
Got it! Just check the discussion. That's a good point! Also, I find that the note text for imageAccessibility component is also 'Text for visually impaired users.', do we want to update that part as well?

Reply:
Also, for the placeholder text, I think it might be too long, do we want to shorten it a bit?

![Screen Shot 2020-11-20 at 9 15 42 AM](https://user-images.githubusercontent.com/11252886/99829335-0861d600-2b11-11eb-8579-eaed71ecd434.png)

Reply:
"Include any burned-in text."

Reply:
Got it!